contracts/contracts/rem.system/src/delegate_bandwidth.cpp

```diff
@@ -249,17 +249,17 @@ namespace eosiosystem {
          voter_itr = _voters.emplace( voter, [&]( auto& v ) {
             v.owner            = voter;
             v.staked           = total_update.amount;
-            v.vote_mature_time = current_time_point() + vote_mature_period;
+            v.locked_stake_period = current_time_point() + vote_mature_period;
```

Comment:
@AndriiAndrusyshyn could you please rename `vote_mature_period` -> `stake_lock_period` to be consistent with new names